schema.graphql

```diff
@@ -414,16 +426,39 @@ type SaleLookupTable @entity {
   sale: Sale!
 }
 
-type Transfer @entity(immutable: true){
+type Transfer @entity(immutable: true) {
   id: ID!
 
   transactionHash: Bytes!
 
-  token: Token! 
+  token: Token!
 
   createdAt: BigInt!
 
   to: Bytes!
 
   from: Bytes!
-}
\ No newline at end of file
+}
+
+enum ProjectExternalAssetDependencyType {
+  "Asset hosted on IPFS"
+  IPFS
+  "Asset hosted on Arweave"
+  ARWEAVE
+}
+type ProjectExternalAssetDependency @entity {
+  "Unique identifier made up of projectId-index"
+  id: ID!
+
+  "The associated project"
+  project: Project!
+
+  "The dependency type"
+  dependencyType: ProjectExternalAssetDependencyType!
+
+  "The dependency cid"
+  cid: String!
+
+  "The dependency index"
+  index: BigInt!
+}
```

Comment:
ya I think it couldn't hurt (to distinguish from updatedAt at the project level, which may be because of an external assets dep operation but may not be)

Reply:
no strong opinion on this, but thinking more on it, `updatedAt` would seem most useful for me tracking against all externalAssetDep entities, vs each individual one.  Are there any use-cases you can think of of that perhaps I'm missing?

Reply:
No strong opinions from me here 👍 just didn't want to forget it if you wanted it 👍